src/test/resources/pages/godmode.html

```diff
@@ -6,10 +6,7 @@
     <div id="mainContent">
         <input type="text" value="">
         <a class="student-profile-pic-view-link btn-link" 
-           data-link="/page/studentProfilePic?studentemail=1CAAAF9A82CEF76B0BA249598DDF9CBC2031E84F1AF45D607A84A83D181C168E&courseid=FA95864E467A11325349CB53B4C6D3C39398C035D135466658ABD102B08D4487">
-           View Photo</a>
-        <a class="student-profile-pic-view-link btn-link" 
-           data-link="/page/studentProfilePic?courseid=FA95864E467A11325349CB53B4C6D3C39398C035D135466658ABD102B08D4487&studentemail=1CAAAF9A82CEF76B0BA249598DDF9CBC2031E84F1AF45D607A84A83D181C168E">
```

Comment:
This test case was there to check the reversal in parameters in some browsers. I think chrome/firefox have different ways to order the parameters.